examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>Don't we define the <code>biharmonic</code> method in differentiable?</p><p> </p><p>Not necessarily for this PR, but with a couple of appropriate sympy functions deriving expressions of arbitrary order could be 'automated'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>